R/importWQP.R

```diff
@@ -117,8 +113,6 @@ importWQP <- function(obs_url, zip=FALSE, tz=""){
                                         `HUCEightDigitCode` = col_character()),
                        quote = "", delim = "\t"))
     
-  if(zip) unlink(doc)
```

Comment:
It is unzipped L63, that zip is removed L64, and the extracted document is removed after the function returns.